style/style.go

```diff
@@ -3,18 +3,18 @@ package style
 import (
 	"fmt"
 
-	"github.com/fatih/color"
+	"github.com/heroku/color"
 )
 
 var Noop = func(format string, a ...interface{}) string {
 	return color.WhiteString("") + fmt.Sprintf(format, a...)
 }
 
 var Symbol = func(format string, a ...interface{}) string {
-	if color.NoColor {
-		format = fmt.Sprintf("'%s'", format)
+	if color.Enabled() {
+		return Key(format, a...)
 	}
-	return Key(format, a...)
+	return fmt.Sprintf("'%s'", Key(format, a...))
```

Comment:
This should not include a call to `Key()`. I suppose color might just get ignored here, but the intention is clearer without the call